src/capture/pcap_service.c

```diff
@@ -112,16 +112,13 @@ int run_pcap(char *interface, bool immediate, bool promiscuous, int timeout,
   struct bpf_program fp;
   struct pcap_context *ctx = NULL;
 
-  if (!find_device(interface, &net, &mask)) {
-    log_error("find_interfaces fail");
-    return -1;
+  if (find_device(interface, &net, &mask) > 0) {
```

Comment:
Do we need to check for error if `find_device(interface, &net, &mask) < 0`?
If no, maybe we should add a comment saying that we're ignoring the error.

Reply:
Added comment.